### PR TITLE
Ignore unsupported type in Phoenix connector

### LINKED ISSUE #2416: Phoenix connector throw unknown connector error when table has unsupported column type
Perhaps, this issue was introduced in https://github.com/prestosql/presto/pull/1182

### PR DESCRIPTION
**Pull request by ebyhr**
Fixes #2416

I suppose this PR is nearly a workaround. If there's a better way to fix it, I would apply it.

### CODE REVIEW

File: presto-phoenix/src/main/java/io/prestosql/plugin/phoenix/PhoenixClient.java [hunk 1]
```diff
@@ -235,6 +237,12 @@ protected String getTableSchemaName(ResultSet resultSet)
         return firstNonNull(resultSet.getString("TABLE_SCHEM"), DEFAULT_SCHEMA);
     }
 
+    @Override
+    public UnsupportedTypeHandling getUnsupportedTypeHandling(ConnectorSession session)
+    {
+        return IGNORE;
```
Comment (kokosing) [inline presto-phoenix/src/main/java/io/prestosql/plugin/phoenix/PhoenixClient.java]: I don't like it. User is allowed to set this catalog session property while it is not used (ignored). I would fix this issue on guice level. To not to use session base jdbc session properties for phoenix at all.

Comment (kokosing) [reply presto-phoenix/src/main/java/io/prestosql/plugin/phoenix/PhoenixClient.java]: Sorry about that. I checked the code and I see that base jdbc session properties are not declared for phoenix. 

I am afraid that jdbc will get more session properties in near future and will hit similar issues like this one.

Comment (ebyhr) [reply presto-phoenix/src/main/java/io/prestosql/plugin/phoenix/PhoenixClient.java]: > I am afraid that jdbc will get more session properties in near future and will hit similar issues like this one.

I think so too. Actually, I tried to bind or isolate Phoenix connector from base jdbc, but I couldn't find the better way and settled on this workaround to resolve regression.

**Comment by kokosing** [comment]
Please see: https://github.com/prestosql/presto/pull/2427

**Pull request closed without merging**